src/main/java/org/cbioportal/web/util/TokenUtils.java

```diff
@@ -0,0 +1,33 @@
+package org.cbioportal.web.util;
+
+import org.cbioportal.service.util.MskWholeSlideViewerTokenGenerator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TokenUtils {
+    @Autowired
+    private Environment env;
+    public String getMskWholeSlideViewerToken() {
```

Comment:
This should be a true util class.

So I would make this into a static function and pass the values in via function arg.

We should move away from the convention of having util classes as components